### PR TITLE
izrss: init at 0.0.5

### PR DESCRIPTION
**Pull request by luftmensch-luftmensch**
## Description of changes
[izrss](https://github.com/isabelroses/izrss) is an RSS feed reader for the terminal written in Go

## Things done

- Built on platform(s)
  - [X] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- For non-Linux: Is sandboxing enabled in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
  - [ ] `sandbox = relaxed`
  - [ ] `sandbox = true`
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [X] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [24.05 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2405.section.md) (or backporting [23.05](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2305.section.md) and [23.11](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2311.section.md) Release notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [X] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

---

Add a :+1: [reaction] to [pull requests you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[pull requests you find important]: https://github.com/NixOS/nixpkgs/pulls?q=is%3Aopen+sort%3Areactions-%2B1-desc

### CODE REVIEW

**Comment by luftmensch-luftmensch** [comment]
```
> nix build .#izrss
> tree result/
 result
└──  bin
   └──  izrss
```

```
./result/bin/izrss --help                                                                                                                                        0.004s 
NAME:
   izrss - Read your favorite news stories from the terminal

USAGE:
   izrss [global options] command [command options]

VERSION:
   unstable

AUTHOR:
   Isabel Roses <isabel@isabelroses.com>

COMMANDS:
   help, h  Shows a list of commands or help for one command

GLOBAL OPTIONS:
   --help, -h     show help
   --version, -v  print the version

CUSTOMIZATION:
    You can customise the colours using "GLAMOUR_STYLE" for a good example see https://github.com/catppuccin/glamour⏎ ```

**Review by isabelroses — APPROVED** [review]
Do you mind adding me as a maintainer, otherwise this seems good to go.

**Comment by luftmensch-luftmensch** [comment]
Ye sure, I didn't had you as maintainer from the beginning as I didn't know if you are willing to maintain it

**Review by isabelroses — APPROVED** [review]

**Comment by isabelroses** [comment]
> I didn't know if you are willing to maintain it

That's understandable. And thank you for adding me.

File: pkgs/by-name/iz/izrss/package.nix [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+    "-w"
+    "-X main.version=${version}"
+  ];
+
+  vendorHash = "sha256-gH5AFroreBD0tQmT99Bmo2pAdPkiPWUNGsmKX4p3/JA=";
+
+  meta = with lib; {
+    description = "An RSS feed reader for the terminal written in Go";
+    homepage = "https://github.com/isabelroses/izrss";
+    license = licenses.agpl3Plus;
```
Comment (isabelroses) [inline pkgs/by-name/iz/izrss/package.nix:32]: Ops. Sorry. I know I just approved but the licence is GPLv3+ not the agpl.

Comment (luftmensch-luftmensch) [reply pkgs/by-name/iz/izrss/package.nix]: Oh I missed that!

**Review by isabelroses — APPROVED** [review]

**Comment by uncenter** [comment]
Result of `nixpkgs-review pr 311889` run on aarch64-darwin [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built:</summary>
  <ul>
    <li>izrss</li>
  </ul>
</details>

**Review by uncenter — CHANGES_REQUESTED** [review]
I don't think the version should be "unstable"?
```
$ ./results/izrss/bin/izrss --version
izrss version unstable
```

**Comment by isabelroses** [comment]
Yeah, just learnt that I need to use var, instead of const.... Oh well, should be fixed in 0.0.6 whenever I release that.

**Comment by luftmensch-luftmensch** [comment]
> Yeah, just learnt that I need to use var, instead of const.... Oh well, should be fixed in 0.0.6 whenever I release that.

@uncenter it all good for know? Or should i patch the line in order to show the correct version?

**Review by uncenter — APPROVED** [review]

**Review by cafkafk — APPROVED** [review]
IMO I don't think patching this is worthwhile, looks good to me :+1:

**Pull request merged**